tempesta_fw/sock_srv.c

```diff
@@ -119,13 +119,13 @@ tfw_sock_srv_connect_try(TfwSrvConnection *srv_conn)
 	}
 
 	sock_set_flag(sk, SOCK_DBG);
-	tfw_connection_link_sk(&srv_conn->conn, sk);
+	tfw_connection_link_from_sk(conn, sk);
```

Comment:
Nice. This is what I mentioned in 2nd paragraph of comment https://github.com/natsys/tempesta/issues/228#issuecomment-137195351 for issue #228.